flink-java/src/main/java/org/apache/flink/api/java/utils/ParameterTool.java

```diff
@@ -136,15 +157,35 @@ public static ParameterTool fromPropertiesFile(String path) throws IOException {
 		return fromMap((Map)props);
 	}
 
+    /**
+     * Returns {@link ParameterTool} for the given map
+	 *
+     * @param map A map of arguments. Both Key and Value have to be Strings
+     * @return A {@link ParameterTool}
+     */
 	public static ParameterTool fromMap(Map<String, String> map) {
 		Preconditions.checkNotNull(map, "Unable to initialize from empty map");
 		return new ParameterTool(map);
 	}
 
+    /**
+     * Returns {@link ParameterTool} from the system properties
```

Comment:
Maybe it would make sense to add that people can pass system properties to the jvm using `-Dkey=value`.